src/index.js

```diff
@@ -1,12 +1,13 @@
-import React from 'react'
+import { Container, CropArea, Img } from './styles'
```

Comment:
I'm not a big fan of this import reordering. I prefer to have the more global ones first and then the local imports.

Reply:
Ahh. No problem. My editor automatically sorted them when I saved the file. I'll undo it and push a new commit in a few.